src/fun_lti_provider/lti.py

```diff
@@ -0,0 +1,196 @@
+"""LTI module that supports LTI 1.0.
+"""
+import re
+from typing import Any, Optional
+
+from oauthlib.oauth1 import SignatureOnlyEndpoint
+
+from fun_lti_provider.models import LTIConsumer, LTIPassport
+
+from .launch_params import LaunchParamException, LaunchParams
+from .validator import LTIRequestValidator
+
+
+class LTI:
+    """The LTI object abstracts an LTI launch request.
+
+    It provides properties and methods to inspect the launch request.
+    """
+
+    def __init__(self, request):
+        """Initialize the LTI system.
+
+        Parameters
+        ----------
```

Comment:
Looks like you are mixing docstring styles. I would prefer sticking to the Google style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html